content/components/selectpanel.mdx

```diff
@@ -309,7 +296,7 @@ Items accompanied by a checkmark icon, as opposed to radio buttons or checkboxes
   </Dont>
 </DoDontContainer>
 
-When an additional checkbox is added to the footer, declarative buttons are required, and the items should be accompanied by visually resembling radio buttons. Although they are not technically radio buttons in terms of semantics, using visual radio buttons can effectively convey that the selection process is not yet complete even after making a choice.
+Adding a checkbox to the footer requires declarative buttons, and items should feature visual radio buttons. While not actual radio buttons semantically, this design effectively indicates that the selection process continues after a choice is made.
```

Comment:
(Not related to just this PR) Why is this the case @maximedegreve?

Also, this is only true for single selection, right? not multiple selection (because radio)

Reply:
Yep!